features/fee_calculator/advocate/fixed_fee_calculator.feature

```diff
@@ -85,21 +85,40 @@ Feature: Advocate completes fixed fee page using calculator
     And I select the 'Number of defendants uplift' fixed fee
 
     Then the following fee details should exist:
-      | section | fee_description | rate | hint | help |
-      | fixed | Elected case not proceeded | 194.00 | Number of days | true |
-      | fixed | Number of cases uplift | 38.80 | Number of additional cases | true |
-      | fixed | Number of defendants uplift | 38.80 | Number of additional defendants | true |
+      | section | fee_description             | rate   | hint                            | help |
+      | fixed   | Elected case not proceeded  | 194.00 | Number of days                  | true |
+      | fixed   | Number of cases uplift      | 38.80  | Number of additional cases      | true |
+      | fixed   | Number of defendants uplift | 38.80  | Number of additional defendants | true |
 
     When I select an advocate category of 'QC'
     Then the following fee details should exist:
-      | section | fee_description | rate | hint | help |
-      | fixed | Elected case not proceeded | 194.00 | Number of days | true |
-      | fixed | Number of cases uplift | 38.80 | Number of additional cases | true |
-      | fixed | Number of defendants uplift | 38.80 | Number of additional defendants | true |
+      | section | fee_description             | rate   | hint                            | help |
+      | fixed   | Elected case not proceeded  | 194.00 | Number of days                  | true |
+      | fixed   | Number of cases uplift      | 38.80  | Number of additional cases      | true |
+      | fixed   | Number of defendants uplift | 38.80  | Number of additional defendants | true |
 
     And I eject the VCR cassette
 
     Then I click "Continue" in the claim form
     And I am on the miscellaneous fees page
 
     And all the fixed fees should have their price_calculated values set to true
+
+  Scenario: I create attempt to create a post-CLAIR elected cases not proceeded claim
```

Comment:
The ECNP validation also affects LGFS claims. I think we should also add this test to `/features/fee_calculator/litigator/fixed_fee_calculator.feature`.

(Not a blocker for this PR, we can do it as a separate change if necessary.)